macros/base/start_print.cfg

```diff
@@ -245,9 +233,7 @@ gcode:
 
     # 9 ----- PRINT !!! -------------------------------------------
     # Do a prime line, lower the lighs and start the print
-    {% if status_leds_enabled %}
-        STATUS_LEDS COLOR="PRINTING"
-    {% endif %}
```

Comment:
Can you put this test back ? it's mandatory to do it here...

As the file `status_leds.cfg` **is optionally included**, it's necessary to do the test **before** calling `STATUS_LEDS COLOR=xxx` to avoid an error about a non-existing macro.

Reply:
Ok done